uk.co.compucorp.civicrm.hrcontactactionsmenu/scss/components/_contact-actions.scss

```diff
@@ -0,0 +1,54 @@
+.crm_contact-actions {
+  background: $gray-light;
+  width: 1065px !important;
```

Comment:
why is the width fixed? Shouldn't it adapt based on the content

Reply:
I checked with @jon it is fixed size.

Reply:
but `.crm_contact-actions` is the entire panel, no? if that's the case, how can it have a fixed width if the content is dynamic?

Reply:
According to @jon, the contact actions panel was intended to be fixed as the panel will contain 4 columns of data regardless of content. And height is auto adjustable based on available content.